index.html

```diff
@@ -5,11 +5,20 @@
 		<link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Open+Sans" />
 	</head>
 	<body>
-		<div class="header">
-			<h1>Westie World!</h1>
+		<div class="header" height="400px" width="400px">
```

Comment:
I'm not sure if putting this height and width here is doing anything

Reply:
Is this an earlier version? Because this doesn't match my most recent commit.

Reply:
Interesting, I'm not sure. It's what GitHub thinks you've added at least :P